### PR TITLE
display toaster upon profile form save success and error

### PR DESCRIPTION
**Pull request by AnthonyBuisset**
closes E-200

### CODE REVIEW

File: frontend/src/hooks/useToaster/useToaster.test.tsx [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+import { ToasterProvider, useToaster } from "./useToaster";
+
+const renderWithProvider = () => renderHook(() => useToaster(), { wrapper: ToasterProvider });
+
+describe("Toaster component", () => {
+  it("should be visible for a given period of time", async () => {
+    const { result } = renderWithProvider();
+
+    expect(result.current.message).toBeUndefined();
+    result.current.showToaster("Message", { duration: 100 });
```
Comment (Bernardstanislas) [inline frontend/src/hooks/useToaster/useToaster.test.tsx:11]: You can even put realistic values here, and use https://vitest.dev/guide/mocking.html#timers – be ready for 🤯

File: tailwind.config.js [hunk 2]
```diff
@@ -12,6 +12,8 @@ module.exports = {
         "noise-medium": "url('src/assets/img/noise-medium.png')",
         "noise-heavy": "url('src/assets/img/noise-heavy.png')",
         space: "url('src/assets/img/bg.jpeg')",
+        "od-gradient":
```
Comment (Bernardstanislas) [inline tailwind.config.js:15]: can you find a more explicit name? Something like "colorful-gradient"?

File: frontend/src/translations/en.json [hunk 3]
```diff
@@ -151,7 +151,7 @@
     "title": "Seniority"
   },
   "state": {
-    "errorOccured": "Oops, an unexpected error occured",
+    "errorOccured": "An error just occurred, please try again. Contact us at contact@onlydust.xyz if the problem persists",
```
Comment (Bernardstanislas) [inline frontend/src/translations/en.json:154]: Let me create the alias right now 😆

**Review by Bernardstanislas — APPROVED** [review]

**Review by oscarwroche — CHANGES_REQUESTED** [review]

File: frontend/src/hooks/useToaster/useToaster.test.tsx (hunk 1 above)
Comment (AnthonyBuisset) [reply frontend/src/hooks/useToaster/useToaster.test.tsx:11]: I will do it in a dedicated PR as I am facing issues with those

File: frontend/src/pages/Profile/components/ProfileForm.tsx [hunk 4]
```diff
@@ -79,9 +81,22 @@ const ProfileForm: React.FC<PropsType> = ({ user }) => {
     },
   });
   const { handleSubmit } = formMethods;
-  const [updateUser, { data }] = useHasuraMutation(UPDATE_USER_MUTATION, HasuraUserRole.RegisteredUser);
+  const [updateUser, { data, error }] = useHasuraMutation(UPDATE_USER_MUTATION, HasuraUserRole.RegisteredUser, {
+    context: {
+      ignoreGraphQLErrors: true, // tell ApolloWrapper to ignore the errors
```
[the remark this answers is not in the document]
Comment (AnthonyBuisset) [reply frontend/src/pages/Profile/components/ProfileForm.tsx]: done @oscarwroche

**Review by oscarwroche — APPROVED** [review]

**Pull request merged**